dom/dom_c.php

```diff
@@ -1328,6 +1328,9 @@ public function registerNodeClass(
 /**
  * The DOMNodeList class
  * @link https://php.net/manual/en/class.domnodelist.php
+ *
+ * @template-covariant TNode as DOMNode
```

Comment:
Right, I just checked and it is unfortunately possible to get a DOMNameSpaceNode out of a DOMNodeList by using XPath. So this should be `TNode as DOMNode|DOMNameSpaceNode` if possible